openedx/core/djangoapps/discussions/admin.py

```diff
@@ -26,6 +27,9 @@ class DiscussionsConfigurationAdmin(SimpleHistoryAdmin):
         'provider_type',
     )
 
+    def changeform_view(self, request, object_id=None, form_url="", extra_context=None):
```

Comment:
I think we should override the top level, `change_view` instead of `changeform_view`. [change_view eventually calls the changeform_view](https://github.com/django/django/blob/86e13843c2ab510fba1de84588efe7fd03555531/django/contrib/admin/options.py#L1964) which does actual work. So, I think overriding the top level method should be better.